plugin/sampling/strategyprovider/adaptive/options.go

```diff
@@ -40,34 +40,30 @@ const (
 // The abbreviation SPS refers to "samples-per-second", which is the target
 // of the optimization/control implemented by the adaptive sampling.
 type Options struct {
-	// TargetSamplesPerSecond is the global target rate of samples per operation.
-	// TODO implement manual overrides per service/operation.
-	TargetSamplesPerSecond float64 `mapstructure:"target_samples_per_second"`
+	Calculation Calculation `mapstructure:"calculation"`
+	Sampling    Sampling    `mapstructure:"sampling"`
 
-	// DeltaTolerance is the acceptable amount of deviation between the observed and the desired (target)
-	// throughput for an operation, expressed as a ratio. For example, the value of 0.3 (30% deviation)
-	// means that if abs((actual-expected) / expected) < 0.3, then the actual sampling rate is "close enough"
-	// and the system does not need to send an updated sampling probability (the "control signal" u(t)
-	// in the PID Controller terminology) to the sampler in the application.
-	//
-	// Increase this to reduce the amount of fluctuation in the calculated probabilities.
-	DeltaTolerance float64 `mapstructure:"delta_tolerance"`
+	// LeaderLeaseRefreshInterval is the duration to sleep if this processor is elected leader before
+	// attempting to renew the lease on the leader lock. NB. This should be less than FollowerLeaseRefreshInterval
+	// to reduce lock thrashing.
+	LeaderLeaseRefreshInterval time.Duration `mapstructure:"leader_lease_refresh_interval"`
 
-	// CalculationInterval determines how often new probabilities are calculated. E.g. if it is 1 minute,
-	// new sampling probabilities are calculated once a minute and each bucket will contain 1 minute worth
-	// of aggregated throughput data.
-	CalculationInterval time.Duration `mapstructure:"calculation_interval"`
+	// FollowerLeaseRefreshInterval is the duration to sleep if this processor is a follower
+	// (ie. failed to gain the leader lock).
+	FollowerLeaseRefreshInterval time.Duration `mapstructure:"follower_lease_refresh_interval"`
```

Comment:
@yurishkuro do you have any thoughts on how to group these? I was thinking grouping them into a type like `Synchronization`.